src/wp-includes/functions.php

```diff
@@ -7635,3 +7645,118 @@ function is_php_version_compatible( $required ) {
 function wp_fuzzy_number_match( $expected, $actual, $precision = 1 ) {
 	return abs( (float) $expected - (float) $actual ) <= $precision;
 }
+
+/**
+ * Formats attributes string from associative array $attributes to be placed inside a `<script>` tag.
+ *
+ * Automatically injects type attribute if needed.
+ * Used by wp_get_script_tag() and wp_get_inline_script_tag().
+ *
+ * @since 5.6.0
+ *
+ * @param array     $attributes     Optional. Key-value pairs representing `<script>` tag attributes.
+ * @return string  String made of sanitized `<script>` tag attributes.
+ */
+function wp_sanitize_script_attributes( $attributes = array() ) {
+	if ( ! array_key_exists( 'type', $attributes ) && ! is_admin() && ! current_theme_supports( 'html5', 'script' ) ) {
+		$attributes['type'] = 'text/javascript';
+	}
+	/**
+	 * Filters script attributes.
+	 *
+	 * @since 5.6.0
+	 *
+	 * @param array $attributes    Key-value pairs representing `<script>` tag attributes.
+	 */
+	$attributes = apply_filters( 'wp_script_attributes', $attributes );
+
+	$attributes_string = '';
+	// Only the attribute name is added to $attributes_string for entries with a boolean value, and that are true.
+	foreach ( $attributes as $attribute_name => $attribute_value ) {
+		if ( is_bool( $attribute_value ) ) {
+			if ( $attribute_value && '' !== $attribute_name ) {
+				$attributes_string .= ' ' . $attribute_name;
+			}
+		} else {
+			$attributes_string .= sprintf( ' %s="%s"', $attribute_name, esc_attr( $attribute_value ) );
+		}
+	}
+
+	return $attributes_string;
+}
+
+/**
+ * Wraps inline JavaScript in `<script>` tags.
+ *
+ * It is possible to inject attributes in the `<script>` tag via the `wp_script_attributes` filter.
+ * Automatically injects type attribute if needed.
+ *
+ * @since 5.6.0
+ *
+ * @param array     $attributes     Key-value pairs representing `<script>` tag attributes.
```

Comment:
```suggestion
 * @param array $attributes Key-value pairs representing `<script>` tag attributes.
```

Since there is no second param to align with, there's no need for so much space